examples/src/main/java/com/mapbox/navigation/examples/ui/NavigationViewActivity.kt

```diff
@@ -121,8 +121,9 @@ class NavigationViewActivity : AppCompatActivity(), OnNavigationReadyCallback, N
     }
 
     private fun getInitialCameraPosition(): CameraPosition {
+        val originCoordinate = route.routeOptions()?.coordinates()?.get(0)?.coordinates()
         return CameraPosition.Builder()
-            .target(LatLng(37.791503, -122.396255))
+            .target(LatLng(originCoordinate!![1], originCoordinate[0]))
```

Comment:
@Guardiola31337 : this addresses the https://github.com/mapbox/mapbox-navigation-android/pull/2818#discussion_r412374755